src/EnergyPlus/HVACControllers.cc

```diff
@@ -2247,13 +2247,13 @@ namespace HVACControllers {
                     // For temperature and humidity control, after temperature control is converged, check if humidity setpoint is met
                     if (!thisController.HumRatCtrlOverride) {
                         // For humidity control tolerance, always use 0.0001 which is roughly equivalent to a 0.015C change in dewpoint
-                        if (Node(thisController.SensedNode).HumRat > (Node(thisController.SensedNode).HumRatMax + 1.0e-4)) {
+                        if (Node(thisController.SensedNode).HumRat > (Node(thisController.SensedNode).HumRatMax + 1.0e-5)) {
                             // Turn on humdity control and restart controller
                             IsConvergedFlag = false;
                             thisController.HumRatCtrlOverride = true;
                             if (thisController.Action == iReverseAction) {
                                 // Cooling coil controller should always be ReverseAction, but skip this if not
-                                RootFinder::SetupRootFinder(RootFinders(ControlNum), iSlopeDecreasing, iMethodBrent, constant_zero, 1.0e-6, 1.0e-4);
+                                RootFinder::SetupRootFinder(RootFinders(ControlNum), iSlopeDecreasing, iMethodFalsePosition, constant_zero, 1.0e-6, 1.0e-5);
```

Comment:
When TemperaturAndHumidityRatio control switch to humidity, use a tighter control tolerance and a different solution method.

Reply:
Can you summarize the difference between iMethodBrent and iMethodFalsePosition?

Reply:
From the function comments:
```
FalsePositionMethod
        // This function computes the next iterate using the false position method (aka regula falsi).
        // If new iterate does not lie within the lower and upper points then
        // the Bisection method is used instead.

BrentMethod
        // This function computes the next iterate using the Brent's method.
        // If new iterate does not lie within the lower and upper points then
        // the secant method is used instead.
        // Inverse quadratic interpolation using the last 3 best iterates.
        // The next root estimate is x = B + P/Q whereby B is the current best estimate
        // of the root.
```
Switching methods here was a shot in the dark that helped settle things down.  I chose false position (regula falsi) because that's a familiar method from other parts of the code. It may be worth exploring whether a method other than `iMethodBrent` would be a better overall default for Controller:WaterCoil.

Reply:
:+1: thanks for the clarification.